community.html

```diff
@@ -70,17 +70,24 @@ <h4 class="card-title">Blog</h4>
         </p>
 
         <h4 class="card-title">Social Media</h4>
-        <a href="https://twitter.com/feedhenry" class="social">
-          <i class="fa fa-twitter" aria-hidden="true"></i> Twitter
-        </a>
-        <br>
-        <a href="https://www.facebook.com/FeedHenry" class="social">
-          <i class="fa fa-facebook" aria-hidden="true"></i> Facebook
-        </a>
-        <br>
-        <a href="https://www.youtube.com/channel/UCYvKWvC7UoUXyLcb16doD3g" class="social">
-          <i class="fa fa-youtube" aria-hidden="true"></i> YouTube
-        </a>
+        <p>
+          <a href="https://twitter.com/feedhenry" class="social">
+            <i class="fa fa-twitter" aria-hidden="true"></i> Twitter
+          </a>
+          <br>
+          <a href="https://www.facebook.com/FeedHenry" class="social">
+            <i class="fa fa-facebook" aria-hidden="true"></i> Facebook
+          </a>
+          <br>
+          <a href="https://www.youtube.com/channel/UCYvKWvC7UoUXyLcb16doD3g" class="social">
+            <i class="fa fa-youtube" aria-hidden="true"></i> YouTube
+          </a>
+        </p>
+        <h4 class="card-title"><i class="fa fa-stack-overflow" aria-hidden="true"></i>  Stack Overflow</h4>
+        <p>
+          We are monitoring questions tagged <a href="http://stackoverflow.com/questions/tagged/aerogear">aerogear</a>, 
```

Comment:
The only reason was because the tag already exists and AeroGear is the home for UPS and Digger upstream.